### ISSUE TITLE
Add ability to see which rooms go the fastest in a ballot

### ISSUE THREAD

**Issue by Pinpickle** [eric-wieser/caius-rooms#20]
Firstly, thank you @eric-wieser for making this!

It makes sense to set our sights realistically when choosing houses to pick from for our balloting time. This could be implemented to varying degrees of usefulness/complexity but some way to see how popular a room is based on previous ballots would help us set those sights realistically.

Useful implementations could be to:

- Give each room a percentile ranking of popularity. Have a list of rooms ordered by popularity
- Given a position in a particular ballot, receive a list of rooms you will likely be able to choose from

I imagine the complexities of this would come from is the numbers around this stem from multiple ballots. e.g. a room may be the half most popular room, not be chosen for one ballot, but then snagged up very quickly in the next (resulting in a popularity that is too high if calculated naively).

**Comment by eric-wieser** [comment]
I worry that any attempt to do this will be either be self-fulfilling, or self-defeating. People behave differently when they have information about how they should behave

**Comment by jack-fellows** [comment]
Yes we've looked into this kind of thing before but didn't think it would be that helpful, and more likely to sway people in a potentially arbitrary direction when picking a room. It should be fairly clear the rough order rooms go in (Old Courts / St Mike's / Town / Morti).

**Comment by eric-wieser** [comment]
This might be sensible to apply at a per-building level, rather than a per-room level

**Comment by jack-fellows** [comment]
Yes that could be more useful, to give (explicitly) the rough order of the ballot. Would it be wise to select the places used though, as some staircases have very few rooms and so it may be better to pick the parent place (e.g. "Gonville Court") to rank by?
